blueprints/codes/latex_formula.py

```diff
@@ -3,6 +3,106 @@
 from dataclasses import dataclass
 
 
+def max_curly_brackets(*args: str | float) -> str:
+    r"""Return a string which will output: max{arg_1; arg_2; ...; arg_N} in latex and it will also automatically ensure floats are converted to latex
+    text.
+
+    Examples
+    --------
+    >>> max_curly_brackets(1, 2)
+    str(\max \left\{1; 2\right\})
+
+    Parameters
+    ----------
+    args: str
+        The arguments of the max function.
+
+    Returns
+    -------
+    str
+        The latex representation of the max function.
+
+    """
+    arguments = [str(arg) for arg in args]
+    return f"\\max \\left\\{{{'; '.join(arguments)}\\right\\}}"
+
+
+def min_curly_brackets(*args: str | float) -> str:
+    r"""Return a string which will output: min{arg_1; arg_2; ...; arg_N} in latex and it will also automatically ensure floats are converted to latex
+    text.
+
+    Examples
+    --------
+    >>> min_curly_brackets(1, 2)
+    str(\min \left\{1; 2\right\})
+
+    Parameters
+    ----------
+    args: str
+        The arguments of the min function.
+
+    Returns
+    -------
+    str
+        The latex representation of the min function.
+
+    """
+    arguments = [str(arg) for arg in args]
+    return f"\\min \\left\\{{{'; '.join(arguments)}\\right\\}}"
+
+
+def fraction(numerator: str | float, denominator: str | float) -> str:
+    r"""Return a string which will output: \frac{numerator}{denominator} in latex.
+
+    Examples
+    --------
+    >>> fraction(1, 2)
+    str(\frac{1}{2})
+
+    Parameters
+    ----------
+    numerator: str | float
+        The numerator of the fraction.
+    denominator: str | float
+        The denominator of the fraction.
+
+    Returns
+    -------
+    str
+        The latex string
+
+    """
+    return f"\\frac{{{numerator}}}{{{denominator}}}"
+
+
+def conditional(*args: list[float | str]) -> str:
+    r"""Return a string which will output a conditional statement with curly brackets with the given arguments and conditions in latex.
+
+    Examples
+    --------
+    >>> conditional([1, "a > 0"], [2, "a < 0"]
+    str(\left{\matrix{1 & \text{voor }a > 0 \\ 2 & \text{voor }a < 0 }\right.)
+
+    Parameters
+    ----------
+    args: list[float|str]
+        A list of length 2, where the first element is the value and the second value is the condition.
+
+    Returns
+    -------
+    str
+        The latex string
+    """
+    string_parts = []
+    for value, condition in args:
+        string_parts.append(f"{value} & \\text{{voor }}{condition} \\\\ ")
+
+    output = r"\left{\matrix{"
+    output += "".join(string_parts)
+    output = output[:-3]
+    return output + r"}\right."
+
+
```

Comment:
although these functions are very usefull this doesn't follow the "YAGNI" principle. Are you using this in the code? 

IMO we shouldn't implement stuff that we aren't using, only until we actually need it.

@egarciamendez: What do you think?

Reply:
You are right, Enrique created these functions, but they have not been reviewed yet. For convenience, I took the functions, thinking I might need them for this or other formulas. However, it may indeed be better to add them only when they are actually used.

Reply:
Those are actually from Peter. But I agree with Wichard on adding those only when necessary.